operations/research-and-development/quality/qa-workflow/README.md

```diff
@@ -1,3 +1,94 @@
-# QA Workflow
+# General QA Workflow
+Note: Specific teams may develop their own variations on this workflow. This is generally how QA works at Mattermost.
 
-Learn more about QA's workflow and processes.
+### Daily
+1. PR reviews (see below)
+2. Jira tickets (see below)
+3. Investigate and reproduce potential issues from reports in channels
+4. Submit bug tickets as issues arise
+5. Help engage with contributor questions in channels
+
+### Weekly
+1. Participate in team triage / planning meetings
+2. Participate in [QA Guild meeting](https://community.mattermost.com/core/channels/qa-weekly-meetings)
+3. Consider giving a demo in QA meeting and/or R&D meeting
+4. (QA Platform) Rainforest release testing and maintenance
+5. (QA Platform) Cypress webapp release testing and maintenance
+6. (QA Platform) Manual release testing and maintenance
+
+### Monthly
+1. SDET rotation to monitor nightly automated tests and investigate and update as needed (see below)
```

Comment:
Could we spell out this acronym, please?

Reply:
Great feedback, @cwarnermm thanks! Done.